lib/kafka_ex/server_0_p_10_and_later.ex

```diff
@@ -268,13 +245,17 @@ defmodule KafkaEx.Server0P10AndLater do
     {:reply, response, state}
   end
 
-  defp request_to_controller(main_request, state) do
+  defp request_to_controller(main_request, stale_state) do
+    # We need to make sure that the metadata is up to date,
+    # to maximize the chance that the controller hasn't switched
+    # to another broker.
+    state = update_metadata(stale_state)
```

Comment:
I was a little concerned about this impacting performance but it looks like it's only called on create/delete topics, so :+1: